sklearn/decomposition/kernel_pca.py

```diff
@@ -8,7 +8,8 @@
 from scipy.sparse.linalg import eigsh
 
 from ..utils import check_random_state
-from ..utils.validation import check_is_fitted, check_array
+from ..utils.validation import check_is_fitted, check_array, \
```

Comment:
please use parentheses rather than `\` for line continuation

Reply:
ok